randnum.c

```diff
@@ -0,0 +1,31 @@
+/* This program guesses and random number
+ * @author Emma Ladouceur
+ */
+
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+int main(){
+
+	srand(time(0));
+	int r = rand() % 10;
```

Comment:
Ah, you were so close! Your range was off - it output numbers from 0 to 9 instead of 1 to 10. You should have had int r = rand() % 10 + 1; to fix this. Other than this, you did a good job working on figuring this out!